src/components/bottomSheet/bottom-sheet.spec.js

```diff
@@ -2,6 +2,52 @@ describe('$mdBottomSheet service', function() {
   beforeEach(module('material.components.bottomSheet'));
 
   describe('#build()', function() {
+    it('should not close when `clickOutsideToClose == true`', inject(function($mdBottomSheet, $rootElement, $material) {
```

Comment:
I generally wouldn't ask someone to change existing lines, but you can if you want.